src/integration/checkout/services/checkout.service.ts

```diff
@@ -85,8 +92,23 @@ export class CheckoutService {
     return payments.filter((p) => !(new Date(p.requested_on) < since));
   }
 
+  async getRefundedPayments(chargebackList: CheckoutTx[]): Promise<CheckoutPayment[]> {
+    const payments: CheckoutPayment[] = [];
+
+    for (const chargeback of chargebackList) {
+      const payment: CheckoutPayment = await this.checkout.payments.get(chargeback.paymentId);
+      payments.push(payment);
+    }
+
+    return payments;
+  }
```

Comment:
I think we should remove the `Refunded` in the method name. This method has nothing to do with refunds 😉